shoop/utils/i18n.py

```diff
@@ -43,9 +40,19 @@ def get_current_babel_locale():
     return get_babel_locale(locale_string=translation.get_language())
 
 
-def format_home_currency(value, locale=None):
-    value = parse_decimal_string(value)
-    return format_currency(value, currency=settings.SHOOP_HOME_CURRENCY, locale=locale or get_current_babel_locale())
+def format_money(amount, digits=None, widen=0, locale=None):
+    loc = babel.Locale.parse(locale or get_current_babel_locale())
+
+    pattern = loc.currency_formats.get(None).pattern
+
+    # pattern is a formatting string.  Couple examples:
+    # '¤#,##0.00', '#,##0.00\xa0¤', '\u200e¤#,##0.00', and '¤#0.00'
+
+    if digits is not None:
+        pattern = pattern.replace(".00", "." + (digits * "0"))
+    if widen:
```

Comment:
No, it would just add them together, i.e. digits=3, widen=4 -> 7 digits. IMHO there's no need to restrict that even though I wouldn't expect anyone to combine them.